puppet-lint.gemspec

```diff
@@ -33,4 +33,5 @@ Gem::Specification.new do |spec|
   spec.license = 'MIT'
 
   spec.required_ruby_version = Gem::Requirement.new('>= 2.5'.freeze)
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
How do you release with MFA via GHA? I see you have https://github.com/puppetlabs/puppet-lint/blob/main/.github/workflows/release.yml and I question how you achieve this.